### PR TITLE
rgw/rgw_op.cc: Fix error message in rgw_user_get_all_buckets_stats

### PR DESCRIPTION
**Pull request by iliul**
Signed-off-by: iliul <liul.stone@gmail.com>

### CODE REVIEW

**Comment by iliul** [comment]
Jenkins, retest this please.

File: src/rgw/rgw_op.cc [hunk 1]
```diff
@@ -1945,20 +1945,20 @@ void RGWGetUsage::execute()
...
   if (op_ret < 0) {
-    ldout(store->ctx(), 0) << "ERROR: failed to sync user stats: " << dendl;
+    ldout(store->ctx(), 0) << "ERROR: failed to sync user stats" << dendl;
     return;
   }
 
   op_ret = rgw_user_get_all_buckets_stats(store, s->user->user_id, buckets_usage);
   if (op_ret < 0) {
-    cerr << "ERROR: failed to sync user stats: " << std::endl;
+    cerr << "ERROR: failed to get user's buckets stats" << std::endl;
```
Comment (cbodley) [inline src/rgw/rgw_op.cc]: oops, could you change this from `cerr` to `ldout` while you're at it?

**Comment by iliul** [comment]
Jenkins, retest this please

File: src/rgw/rgw_op.cc (hunk 1 above)
Comment (iliul) [reply src/rgw/rgw_op.cc]: @cbodley done！

**Comment by iliul** [comment]
Jenkins, retest this please

**Comment by cbodley** [comment]
jenkins test this please

**Pull request merged**